<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientCard</class>
 <widget class="QWidget" name="PatientCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0">
    <widget class="QLabel" name="resLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Res.date</string>
     </property>
    </widget>
   </item>
   <item row="14" column="3">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>save</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="ageEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="istbolLabel">
     <property name="text">
      <string>IST BOL</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="memoLabel">
     <property name="text">
      <string>Memo</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="QComboBox" name="patientBox"/>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="resdateEdit"/>
   </item>
   <item row="0" column="1" colspan="4">
    <widget class="QComboBox" name="doctorBox"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="doctorLabel">
     <property name="text">
      <string>Doctor</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="ageLabel">
     <property name="text">
      <string>AGE</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="patientLabel">
     <property name="text">
      <string>Patient</string>
     </property>
    </widget>
   </item>
   <item row="14" column="4">
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string>close</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="ifsLabel">
     <property name="text">
      <string>IFS</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLineEdit" name="ifsEdit"/>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="diagLabel">
     <property name="text">
      <string>Diag</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1" colspan="4">
    <widget class="QLineEdit" name="diagEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="12" column="1" colspan="4">
    <widget class="QLineEdit" name="memoEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="istbolEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
